apps/aehttp/priv/swagger.json

```diff
@@ -4093,15 +4093,11 @@
     },
     "UnsignedTx" : {
       "type" : "object",
-      "required" : [ "tx", "tx_hash" ],
```

Comment:
Spurious `, "tx_hash" : null` in `example`?

Reply:
Fixed